app/controllers/uploads_controller.rb

```diff
@@ -91,8 +91,11 @@ def call_load
 
     CrosswalkIssue.rebuild if [Crosswalk, IpedsHd, Weam].include?(klass)
 
-    @upload.update(ok: data.present? && data.ids.present?)
     error_msg = "There was no saved #{klass} data. Please check \"Skip lines before header\" or \"Column separator\"."
+    @upload.update(ok: data.present? && data.ids.present?,
+                   comment: error_msg,
```

Comment:
Looks like you are adding this `error_msg` to every `@upload` whether there was an error or not.

Reply:
removed